.github/workflows/upload.yml

```diff
@@ -27,7 +31,7 @@ jobs:
 
       - name: Run tests
         run: |
-          python -m pytest tests --tb=native
+          python -m pytest tests -p no:warnings --randomly-seed=42 --tb=native
```

Comment:
For future reference I'm attaching this comment from PR #480:

> Optimally, we would remove the `randomly-seed=42` from the tests, causing the seed to be set randomly each time, helping us discover potential bugs due to the stochastic nature of some tests. By resetting the seed before each test, we may miss failures that might be present. This might cause some headaches, though.